docs/tutorial.md

```diff
@@ -18,6 +18,86 @@ The wizard will also ask if you'd like it setup any additional metadata files su
 > Ref: See [`zigmod init`](./commands/init.md) for more info.
 
 ---
+## Understanding `zig.mod`
+
+Running `zigmod init` will ask if You want to create an application or a
+library. Based on You answer the resulting `zig.mod` file have two types.
+
+### `zig.mod` for an application
+
+```yaml
+id: ucst208pgxu7c1slqop2ng9llcjy4369zmupp21wxjvv0nib
```

Comment:
leaving `id:` out here might be good so that users don't accidentally copy/paste this and share the same ID